packages/e2e-tests/tests/highlighter.test.ts

```diff
@@ -33,6 +33,6 @@ test("highlighter: element highlighter works everywhere", async ({
     .join(" ");
 
   // TODO:FE-805 This is a hack to get the test to pass. We should figure out why the path definition is different from the original and not use a hardcoded value.
```

Comment:
Is this TODO comment (still) relevant?

Reply:
Can probably be removed. Don't remember what the old pre-Playwright test was like, and it's not worth digging back at this point.